.ci/docker-package-push.sh

```diff
@@ -25,7 +25,3 @@ make .webhook
 
 echo "INFO: Push docker image ${fqn}"
 docker push "${fqn}"
-
-echo "INFO: Push docker image ${latest}"
-docker tag "${fqn}" "${latest}"
-docker push "${latest}"
```

Comment:
`latest` is not pushed at all anymore with this right?

Reply:
Correct. We have the "latest" already published and we don't want to be doing CD since the helm chart is pointing to `latest`. I can revert this part and we can continue building the "latest" image for now, but it seems out of what we expect.